testsuite/tests/atomic-locs/record_fields.ml

```diff
@@ -0,0 +1,116 @@
+(* TEST
+   flags = "-dlambda -dno-locations -dno-unique-ids";
+   expect;
+*)
+
+(* Check module interface checking: it is not allowed to remove or add
+   atomic attributes. *)
+
+module Wrong1 = (struct
```

Comment:
Is there a reason to use this syntax rather than `module M : sig ... end = struct ... end`? I got the struct and sig backwards on a first reading.